packages/components/psammead-headings/README.md

```diff
@@ -4,14 +4,43 @@
 
 The Headings are a set of two components, `Headline` and `SubHeading`. They use a `h1` and `h2` HTML element respectively, with the `Headline` being designed for a single use on the page, with `SubHeading` being aimed at repeated use.
 
-## When to use this component
+## Installation
+
+`npm install @bbc/psammead-headings`
+
+## Props
+
+| Argument  | Type                | Required | Example         |
+|-----------|---------------------|----------|-----------------|
+| No props. |
+
+## Usage
+
+```jsx
+import { Headline, SubHeading } from '@bbc/psammead-headings';
+
+const Wrapper = (src, alt, width, height) => (
+    <Fragment>
+        <Heading>Some headline</Heading>
+        <SubHeading>Some subheadline</SubHeading>
+    </Fragment>
+);
+```
+
+All `SubHeading` components can be used as page anchors, with their ID being generated from their text, with any whitespace replaced with hyphens.
```

Comment:
Can you give an example? I wouldn't have known these were linkable otherwise! Thanks for adding the sentence 👍